qiskit_experiments/curve_analysis/visualization/mpl_drawer.py

```diff
@@ -0,0 +1,361 @@
+
+                if tick_labels:
+                    # Set formatter only when tick labels exist
+                    ax.set_major_formatter(formatter)
+                if units_str:
+                    # Add units to label if both exist
+                    label_txt_obj = ax.get_label()
+                    label_str = label_txt_obj.get_text()
+                    if label_str:
+                        label_txt_obj.set_text(label_str + units_str)
+
+            # Auto-scale all axes to the first sub axis
+            if ax_type == "x":
+                all_axes[0].get_shared_x_axes().join(*all_axes)
+                all_axes[0].set_xlim(lim)
+            else:
+                all_axes[0].get_shared_y_axes().join(*all_axes)
+                all_axes[0].set_ylim(lim)
+
+    def _get_axis(self, index: Optional[int] = None) -> Axes:
+        """A helper method to get inset axis.
+
+        Args:
+            index: Index of inset axis. If nothing is provided, it returns the entire axis.
+
+        Returns:
+            Corresponding axis object.
+
+        Raises:
+            IndexError: When axis index is specified but no inset axis is found.
+        """
+        if index is not None:
+            try:
+                return self._axis.child_axes[index]
+            except IndexError as ex:
+                raise IndexError(
+                    f"Canvas index {index} is out of range. "
+                    f"Only {len(self._axis.child_axes)} subplots are initialized."
+                ) from ex
+        else:
+            return self._axis
+
+    def draw_raw_data(
+        self,
+        x_data: Sequence[float],
+        y_data: Sequence[float],
+        ax_index: Optional[int] = None,
+        **options,
+    ):
+        draw_options = {
+            "color": "grey",
+            "marker": "x",
+            "alpha": 0.8,
+            "zorder": 2,
+        }
+        draw_options.update(**options)
+        self._get_axis(ax_index).scatter(x_data, y_data, **draw_options)
+
+    def draw_formatted_data(
+        self,
+        x_data: Sequence[float],
+        y_data: Sequence[float],
+        y_err_data: Sequence[float],
+        name: Optional[str] = None,
+        ax_index: Optional[int] = None,
+        **options,
+    ):
+        draw_ops = {
+            "markersize": 9,
+            "alpha": 0.8,
+            "zorder": 4,
+            "linestyle": "",
+        }
+        draw_ops.update(**options)
+        if name:
+            draw_ops["label"] = name
+        self._get_axis(ax_index).errorbar(x_data, y_data, yerr=y_err_data, **draw_ops)
+
+    def draw_fit_lines(
+        self,
+        fit_function: Callable,
+        signature: List[str],
+        fit_result: FitData,
+        fixed_params: Dict[str, float],
+        ax_index: Optional[int] = None,
+        **options,
+    ):
+        draw_ops = {
+            "markersize": 9,
+            "zorder": 5,
+            "linestyle": "-",
+            "linewidth": 2,
+        }
```

Comment:
Is there any way to configure these values?

Reply:
Yes. In follow-up you can add arbitrary drawing options to override them. This is indeed one of feedback from the beta-test.